docs/products/access/api-reference/description-oauth-endpoint.md

```diff
@@ -245,3 +245,92 @@ See the OAuth specification for [token revocation](https://tools.ietf.org/html/r
 
 Note for refreshing and revoking the access token: When a public client uses the authorization code grant type with PKCE, the request parameter `client_id` is mandatory.
 For *confidential* clients both the client id and the client secret are sent as basic authentication credentials via the `authorization` header.
+
+## OAuth Device Flow Extension
+
+### Introduction
+
+The OAuth Device Flow is an extension to the standard OAuth 2.0 protocol, specifically designed for devices with no browser or limited input
+capability. This flow enables users to authorize such devices in a user-friendly and secure manner. After entering the correct code, the
+user proceeds through the standard authorization process, including authentication and, if enabled on the client, consent. For detailed
+specifications, refer to [RFC 8628](https://tools.ietf.org/html/rfc8628).
+
+### Authorization Process
+
+The client initiates the authorization process by sending a POST request to the `/oauth/device_authorization` endpoint. This request
+includes the client's unique identifier (`client_id`) and optionally the requested `scope` and identity provider (`idp`). The server
+responds with a `device_code`, `user_code`, and `verification_uri`. The user must visit this URI in a browser and enter the user code to
```

Comment:
Maybe we should give more info about the options that we have for `user_code`, and `verification_uri`